examples/options/include/traccc/options/truth_finding.hpp

```diff
@@ -0,0 +1,23 @@
+/** TRACCC library, part of the ACTS project (R&D line)
+ *
+ * (c) 2023-2024 CERN for the benefit of the ACTS project
```

Comment:
I know there are different schools of thoughts about this, but `2023-2024` doesn't seem to fit either of those to me. 😛